### DISCUSSION TITLE
Could not resolve to a user with the user of...

### DISCUSSION THREAD

**Discussion by yagueto** [community/community#21136] [Mobile] [answered]
In the app, when I go to a repo and and tap to add it to a list, I get the error "Could not resolve to a user with the user of Yagueteiro".

Might it have to do with the fact that "yagueteiro" was my username before?

**Comment by aconsuegra** [comment, marked as answer]
Hi @yagueto, thanks for the bug report. I think you are correct and changing your user name could have created some issues. Do you mind logging out from the app, and trying to add a repo to your lists after you login again?

**Reply by yagueto** [reply]
Yep, that solved it, thanks
